jupyter_dash/jupyter_app.py

```diff
@@ -149,6 +149,7 @@ def alive():
     def run_server(
             self,
             mode=None, width="100%", height=650, inline_exceptions=None,
+            jupyterlab_title="",
```

Comment:
@litghost apologies for losing track of this PR for so long. If you're still interested in this, it's a great start, just two comments:
- we can call this kwarg just `title`, since we already have examples of kwargs here (`width` and `height`) that apply only to one mode without including the mode in the name.
- let's add this to the docstring below, and to the changelog.

If you've moved on and don't want to come back to this, no worries, I'll get this across the finish line.